### DISCUSSION TITLE
Erro "Unhandled event loop exception"

### DISCUSSION THREAD

**Discussion by fagnermacedo** [eclipse-birt/birt#1162] [Q&A]
Hi,
 
I'm trying to use Birt 4.10 with JVM 19.
When I try to see the preview of report, the eclipse show a erro.
**java.lang.NullPointerException: Cannot invoke "org.eclipse.swt.widgets.Control.setMenu(org.eclipse.swt.widgets.Menu)" because the return value of "org.eclipse.ui.forms.editor.IFormPage.getPartControl()" is null**

Does anyone know how to solve this problem ?

Thks.

**Comment by speckyspooky** [comment]
Could you please try to use the latest BIRT version 4.12 (eclipse all-in-one-designer). It will be deliverd with JVM 17.
Please start a try with the standard if it works for your report.
and afterwards you can change the birt.ini to your JVM 19 and repeat your test.
So we can compare the results of JVM 17 and JVM 19.

Best regards
